FASTDOOM/ns_music.c

```diff
@@ -34,10 +34,7 @@ int MUSIC_InitGUS(midifuncs *Funcs);
    Selects which sound device to use.
 ---------------------------------------------------------------------*/
 
-int MUSIC_Init(
-    int SoundCard,
-    int Address)
-
+int MUSIC_Init(int SoundCard, int Address)
```

Comment:
I was following the code and I don't see where this address is passed through, 
maybe this is the issue?

Reply:
Yep, I think that was the issue. I've uploaded another commit to fix it. Can you try again?